File: src/main/java/com/jrmplugin/exception/TaskIdIncorrectException.java
package com.jrmplugin.exception;

public class TaskIdIncorrectException extends RuntimeException {

    public static final String TASK_ID_HAS_NOT_CHANGED = "Please put correct task id to the form.";
    public static final String TASK_ID_IS_NOT_UUID = "Incorrect task id = ";

    public TaskIdIncorrectException(String message) {
        super(message);
    }

    public TaskIdIncorrectException(String message, String taskId) {
        super(message + taskId);
    }

}

File: src/main/java/com/jrmplugin/exception/TaskIdIncorrectException.kt
package com.jrmplugin.exception

class TaskIdIncorrectException : RuntimeException {
    constructor(message: String?) : super(message) {}
    constructor(message: String, taskId: String) : super(message + taskId) {}

    companion object {
        const val TASK_ID_HAS_NOT_CHANGED = "Please put correct task id to the form."
        const val TASK_ID_IS_NOT_UUID = "Incorrect task id = "
    }
}